x/cert/utils/utils.go

```diff
@@ -0,0 +1,90 @@
+package utils
+
+import (
+	"crypto/ecdsa"
+	"crypto/tls"
+	"crypto/x509"
+	"encoding/pem"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"reflect"
+
+	"github.com/cosmos/cosmos-sdk/crypto/keyring"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/pkg/errors"
+
+	ctypes "github.com/ovrclk/akash/x/cert/types"
+)
+
+// LoadPEMFromFrom load certificate/private key from file named as
+// account name supplied in FlagFrom
+// file must contain two PEM blocks, certificate followed by a private key
+func LoadPEMFromFrom(homedir string, key sdk.Address, keyring keyring.Keyring) ([]byte, []byte, []byte, error) {
```

Comment:
It seems like it would be easier to return some sort of struct rather than 3 byte values here.